models/articles-models.js

```diff
@@ -33,14 +33,19 @@ exports.fetchArticleComments = (article_id) => {
 
 exports.fetchArticleById = (article_id) => {
   const articleQueryString = `
-SELECT article_id, author, topic, title, body,
-created_at, votes, article_img_url
-FROM articles
-WHERE article_id = $1;`;
+SELECT articles.article_id, articles.author, articles.topic, articles.title, 
+articles.body, articles.created_at, articles.votes, articles.article_img_url, 
+COUNT(comments.comment_id) AS comment_count
+FROM articles FULL OUTER JOIN comments ON comments.article_id = articles.article_id
```

Comment:
This join will work in this situation because the WHERE clause means that we can't select any comments without a corresponding article. However, the problem seems to lend itself more to a different type of join.